src/main/resources/swagger/schemas/matches-json-schema-pattern.yaml

```diff
@@ -3,23 +3,40 @@ type: object
 properties:
   matchesJsonSchema:
     oneOf:
```

Comment:
I don't think this `oneOf` is needed at all, could be removed.

Reply:
Removed